### ISSUE TITLE
Missing highlight if HEAD on commit

### ISSUE THREAD

**Issue by bboehmke** [mhutchie/vscode-git-graph#18]
**Describe the bug**
If a specific commit is used for checkout it is not highlighted in the Graph.

**What does the bug relate to**
Graph Rendering

**To reproduce**
Checkout a commit that is not on the end of a branch.

**Expected behaviour**
The same highlight as for the checkout on a branch end.

**Environment**
 - OS: Windows 10
 - Visual Studio Code Version: 1.31.1
 - Plugin Version: 1.3.3

**Screenshots (optional)**
![image](https://user-images.githubusercontent.com/9787279/53558086-98930700-3b47-11e9-9ce1-b3c70d50b9a0.png)

```
$ git status
HEAD detached at afd32c6
nothing to commit, working tree clean
```

**Comment by mhutchie** [comment]
Thanks for picking this up, and giving a great example for me to reference! I'll have this fixed for tomorrows release of 1.4.0.

**Comment by mhutchie** [comment]
This has been fixed, and will be available in v1.4.0

**Comment by bboehmke** [comment]
With version 1.4.0 it works fine. 
Thanks for the fast fix.
